### DISCUSSION TITLE
Suppression comment is not being used

### DISCUSSION THREAD

**Discussion by igortas** [biomejs/biome#4161] [Q&A]
Hi,

I'm using the unplugin for autoimport, and biome gives me warning for the autogenerated file.
The lint option for surprass is added, but still shows warning...

![image](https://github.com/user-attachments/assets/f4d44320-7a20-43f5-9b3c-917b3230e3e8)

**Comment by ematipico** [comment]
Have you tried to remove the suppression comment? If you do so, do you get another diagnostic instead?

**Comment by igortas** [comment]
Hi @ematipico 
Removing the suppression comment, unplugin detects biome and adds it on it's own...

![{84C69B6F-5741-4904-9E35-A38668BC7B9D}](https://github.com/user-attachments/assets/46a8b841-4d97-44b0-a724-b5953c539f49)

**Reply by ematipico** [reply]
I really don't understand what you're trying to do, and I don't understand the unplugin part. You should ask the people that generated that code for you what it is the way it is.

**Comment by igortas** [comment]
@ematipico biome giving the warning on lint step, the plugin adds the comments...so biome is not ignorning  the file...

the file is also in ignore list...
![{CB8B9440-DE00-4238-9BEC-56EBAE8EBCCF}](https://github.com/user-attachments/assets/2a36b126-102f-4ab4-8532-5718afadae40)
